plugins/snapshots/windows/cimfs.go

```diff
@@ -327,27 +325,37 @@ func (s *cimFSSnapshotter) mounts(sn storage.Snapshot, key string) []mount.Mount
 
 	source := s.getSnapshotDir(sn.ID)
 	parentLayerPaths := s.parentIDsToParentPaths(sn.ParentIDs)
+	layerCimPaths := s.parentIDsToCimPaths(sn.ParentIDs)
 
 	mountType := "CimFS"
 
 	// error is not checked here, as a string array will never fail to Marshal
 	parentLayersJSON, _ := json.Marshal(parentLayerPaths)
 	parentLayersOption := mount.ParentLayerPathsFlag + string(parentLayersJSON)
+	parentCimLayersJSON, _ := json.Marshal(layerCimPaths)
+	parentCimLayersOption := ParentLayerCimPathsFlag + string(parentCimLayersJSON)
```

Comment:
Can we make the variable names consistent here between paths/cims?

e.g.

`parentCimPaths`, `parentCimsJSON`, `parentCimsOption`